packages/nextcloud/lib/src/api/files_sharing.openapi.json

```diff
@@ -2793,26 +2801,20 @@
                         "in": "query",
                         "description": "Limit to specific share types",
                         "schema": {
-                            "type": "string",
                             "nullable": true,
-                            "contentMediaType": "application/json",
-                            "contentSchema": {
-                                "nullable": true,
-                                "description": "Limit to specific share types",
-                                "oneOf": [
-                                    {
+                            "oneOf": [
+                                {
+                                    "type": "integer",
+                                    "format": "int64"
+                                },
+                                {
+                                    "type": "array",
+                                    "items": {
                                         "type": "integer",
                                         "format": "int64"
-                                    },
-                                    {
-                                        "type": "array",
-                                        "items": {
-                                            "type": "integer",
-                                            "format": "int64"
-                                        }
                                     }
-                                ]
-                            }
+                                }
+                            ]
```

Comment:
I'd like a unit test on that one.
It should not be affected by #1058 but I'd like to have it tested.

Reply:
Nope this is exactly https://github.com/nextcloud/neon/pull/1215 :/